### ISSUE TITLE
[A11y] Search Results - Grid - Labels

### ISSUE THREAD

**Issue by jechols** [OregonDigital/OD2#1919]
### Operating System

Windows

### Browser

Edge

### Affected page

Search

### Admin user?

Yes

### Issue Description

The checkboxes on the various search results have no label.

### Suggested Fix

I'm not sure what a good label might be for these. The purpose of the checkbox seems to be for adding items to a collection, but that action is a ways away from the checkboxes. If the checkbox label is too long (e.g., "Select A11y Test 6 for adding to a custom collection using the SAVE SELECTED TO button"), that arguably makes the checkboxes only slightly better than they already are.

Announcing something like "Select A11y Test 6" might be the best choice, though it certainly isn't clear what they're selected for.

### Can anybody QA or does this need an expert?

Needs expert

### QA Steps

This is best tested in a screen reader to figure out if the context of what's read makes sense.  Barring that, using Firefox one could inspect the accessibility properties and try to gauge if the checkboxes' accessible names are effectively communicating their meaning without being unnecessarily verbose.

**Comment by CGillen** [comment]
Fixed by #2059
